README.md

```diff
@@ -24,18 +24,29 @@ their awesome layout in their Discover feed.
   * Magazines
   * News articles
 
-## Tutorial
+## Installation
+
+#### CocoaPods
+You can use [CocoaPods](http://cocoapods.org/) to install `TRMosaicLayout` by adding it to your `Podfile`:
 
-Add `TRMosaicLayout` to your podfile
 ```ruby
+platform :ios, '8.0'
+use_frameworks!
 pod 'TRMosaicLayout'
 ```
 
-Then, run in your terminal
-```bash
-$ pod install 
+To get the full benefits import `TRMosaicLayout` wherever you import UIKit
+
+``` swift
+import UIKit
+import YourLibrary
```

Comment:
Could you update this to be `import TRMosaicLayout` instead of `import YourLibrary`